presentation/src/main/java/me/androidbox/presentation/agenda/screen/AgendaScreen.kt

```diff
@@ -33,7 +33,9 @@ import me.androidbox.presentation.ui.theme.BusbyTaskyTheme
 fun AgendaScreen(
     agendaScreenState: AgendaScreenState,
     agendaScreenEvent: (AgendaScreenEvent) -> Unit,
+    onSelectedAgendaItem: (agendaType: Int) -> Unit, /* TODO Check where this is being used */
```

Comment:
I'd pass the agendatype and not the int

Then have the MainActivity handle it how it wants

Reply:
@MKiperszmid Will update this, I left a TODO just in case I forget :blush: